### PR TITLE
Improve post release tasks in workflows

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -45,19 +45,9 @@ jobs:
...
-          curl --request POST 'https://api.github.com/repos/ballerina-platform/module-ballerina-sql/releases' \
-          --header 'Accept: application/vnd.github.v3+json' \
-          --header 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
-          --header 'Content-Type: application/json' \
-          --data-raw '{
-              "tag_name": "v'"$VERSION"'",
-              "name": "module-ballerina-sql-v'"$VERSION"'"
-          }'
-      - name: Post release PR
+      - name: Post release tasks
```
Comment (ThisaruGuruge) [inline .github/workflows/publish-release.yml:48]: This is not exactly `Post Release Tasks` it seems? May be `Github Release`?

Comment (niveathika) [reply .github/workflows/publish-release.yml:48]: Github Release and Release sync PR?

Comment (ThisaruGuruge) [reply .github/workflows/publish-release.yml:48]: +1

Comment (niveathika) [reply .github/workflows/publish-release.yml]: Fixed in efafc55

File: .github/workflows/publish-release.yml [hunk 2]
```diff
@@ -45,19 +45,9 @@ jobs:
...
-          curl --request POST 'https://api.github.com/repos/ballerina-platform/module-ballerina-sql/releases' \
-          --header 'Accept: application/vnd.github.v3+json' \
-          --header 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
-          --header 'Content-Type: application/json' \
-          --data-raw '{
-              "tag_name": "v'"$VERSION"'",
-              "name": "module-ballerina-sql-v'"$VERSION"'"
-          }'
-      - name: Post release PR
+      - name: Github Release and Release sync PR
```
[the remark this answers is not in the document]
Comment (BuddhiWathsala) [reply .github/workflows/publish-release.yml:48]: Change it to the title case.

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**